http-servers/src/main/java/com/palantir/remoting/http/server/NotFoundExceptionMapper.java

```diff
@@ -1,30 +0,0 @@
-/*
- * Copyright 2015 Palantir Technologies, Inc. All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.palantir.remoting.http.server;
-
-import javax.ws.rs.NotFoundException;
-import javax.ws.rs.core.Response.Status;
-import javax.ws.rs.ext.Provider;
-
-@Provider
-public final class NotFoundExceptionMapper extends JsonExceptionMapper<NotFoundException> {
```

Comment:
Yeah, the different specific web exceptions bake their ``Status`` in. (I can add a test for ``ForbiddenException`` if you want to be super certain that we're covering all the same things as before, but I feel like we don't want the mess of confirming that every specific exception actually encodes what it claims to encode)